### PR TITLE
Server

### PR DESCRIPTION
**Pull request by xueccc**
Set up server

### CODE REVIEW

**Comment by xueccc** [comment]
Hi @ChuaJ , would you please review my PR?

File: client/src/index.jsx [hunk 1]
```diff
@@ -1,8 +1,20 @@
-var component = function() {
-  var element = document.createElement('div');
-  element.innerHTML = _.join(['Shop', 'Prodcuts'], ' ');
-  return element;
-}
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+var axios = require('axios');
```
Comment (ChuaJ) [inline client/src/index.jsx]: You also want to have this be an import. Similar to how we've imported jquery before: import $ from "jquery".

File: client/src/index.jsx [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
+  render() {
+    return (
+      <div>
+        Shop modules
+      </div>
+    )
+  }
+}
 
+ReactDOM.render(<App />, document.getElementById('app'));
```
Comment (ChuaJ) [inline client/src/index.jsx]: Something to consider, we should all probably anchor our components to different IDs. Let the group know you're using "app"!

File: server/index.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+var express = require('express');
+var app = express();
+app.use(express.static(__dirname + '/../public/dist'));
+
+app.get('/', function(req, res){
+  res.send()
+});
+
+
+app.listen(5000, ()=>console.log('App listening to port 5000'));
```
Comment (ChuaJ) [inline server/index.js]: New line!

**Pull request closed without merging**